extensions/scarb-doc/src/docs_generation/markdown/traits.rs

```diff
@@ -126,49 +158,60 @@ impl MarkdownDocItem for Module {
             &self.submodules.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.constants.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.free_functions.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.structs.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.enums.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.type_aliases.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.impl_aliases.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.traits.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.impls.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.extern_types.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
         )?;
-        markdown += &generate_markdown_list_for_top_level_subitems(
+        markdown += &generate_markdown_table_summary_for_top_level_subitems(
             &self.extern_functions.iter().collect_vec(),
-            header_level + 1,
+            context,
+            false,
```

Comment:
imo, for consistency sake it's good to have all item types in tables instead of modules-in-tables the-rest-in-lists but I do hesitate and don't have any strong feelings on that. 
I guess that quite subjective and subjected to the end-user perspective. Any opinion on that?